pkg/spec/spec.go

```diff
@@ -72,6 +72,7 @@ type ConfigMapMod struct {
 
 type PodSpecMod struct {
 	Containers     []Container `json:"containers,omitempty"`
+	InitContainers []Container `json:"initContainers,omitempty"`
```

Comment:
Ah, I understand now. 

Shouldn't we break this PR to two? How is moving populators to separate file related to adding initContainers?